src/ontology/cl-edit.owl

```diff
@@ -7089,10 +7089,13 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000440 "melanotroph")
 AnnotationAssertion(rdfs:label obo:CL_0000440 "melanocyte stimulating hormone secreting cell")
 EquivalentClasses(obo:CL_0000440 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0036160)))
 
-# Class: obo:CL_0000441 (follicle stem cell)
+# Class: obo:CL_0000441 (follicle stem cell (sensu Arthropoda))
 
-AnnotationAssertion(rdfs:label obo:CL_0000441 "follicle stem cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/B978-012369493-5.50005-5") Annotation(oboInOwl:hasDbXref "doi:10.1242/dev.121.11.3797") obo:IAO_0000115 obo:CL_0000441 "A stem cell that gives rise to the follicle cells that surround the oocyte in female arthropods.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.devcel.2005.08.012") oboInOwl:hasBroadSynonym obo:CL_0000441 "somatic stem cell")
```

Comment:
Currently “somatic stem cell“ is a subclass of [epithelial fate stem cell](http://purl.obolibrary.org/obo/CL_0000036), which is correct and more precise than [somatic stem cell](http://purl.obolibrary.org/obo/CL_0000723), which is higher up in the hierarchy.

The synonym is there because “somatic stem cell” is frequently used to refer to these cells, but that name is precise enough _only_ when it is used in the context of the insect ovary, where it is used in opposition to “germline stem cells” (the other type of stem cells in the ovary).

The “escort stem cells” mentioned in one of the papers refer to yet another type of cell (one whose existence is actually dubious).